src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -558,7 +558,6 @@ void ResumeCtrlImpl::ResetLaunchTime() {
 bool ResumeCtrlImpl::CheckPersistenceFilesForResumption(
     ApplicationSharedPtr application) {
   SDL_LOG_AUTO_TRACE();
-  DCHECK_OR_RETURN(application, false);
```

Comment:
Not certain whether we need to remove these checks, I realize that they might be slightly overkill, but realistically they don't hurt to have

Reply:
restored in https://github.com/smartdevicelink/sdl_core/pull/3815/commits/bed8d7af1592023f3ce8578754c5d96b0b8bbe4a